shared/teams/team/settings-tab/default-channels.tsx

```diff
@@ -43,6 +38,20 @@ const DefaultChannels = (props: Props) => {
   // Initialize
   React.useEffect(reloadDefaultChannels, [reloadDefaultChannels])
 
+  return {defaultChannels, defaultChannelsWaiting, error, reloadDefaultChannels}
+}
+
+const DefaultChannels = (props: Props) => {
+  const {teamID} = props
+  const {defaultChannels, defaultChannelsWaiting, reloadDefaultChannels} = useDefaultChannels(teamID)
+  const setDefaultChannelsRPC = Container.useRPC(RPCChatGen.localSetDefaultTeamChannelsLocalRpcPromise)
+  const [waiting, setWaiting] = React.useState(false)
+  // TODO TRIAGE-2474
+  // Implicit admins should be able to set this, but chat stuff doesnt know about them.
+  // For now limit to people who are admins in this team.
+  // const canEdit = Container.useSelector(s => Constants.getCanPerformByID(s, teamID).manageMembers)
+  const canEdit = Container.useSelector(s => ['admin', 'owner'].includes(Constants.getRole(s, teamID)))
+
   const onAdd = (channels: Array<Types.ChannelNameID>) => {
```

Comment:
I was unable to add a channel here. Got a forever spinner. Unless it's a 2-GUI issue, wanna investigate?

Reply:
So, the display didn't work for you at all? I haven't seen that. Since it uses the chat RPC i'm guessing a 2-GUI issue is likely.

Reply:
it showed, but when I tried to add a channel I got a forever spinner. If it works for you I assume it's a 2-GUI issue since chat RPCs, yeah.

Reply:
Huhh, on this confirm screen? Not the team settings tab, right? This should just be changing the store on adding, no rpcs. What actions happen when you try to add?

Reply:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/4698687/78158045-c57eb100-740e-11ea-8c96-011bb0b8a7ff.png">
works fine on the confirm screen. This is the settings tab

Reply:
Oh I forgot there were changes here. Works for me in settings. It could be an outdated service or yeah 2 guis.